<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>240</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans Condensed SemiBold</family>
    <pointsize>14</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>EditWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(233, 234, 255)</string>
  </property>
  <widget class="QDialogButtonBox" name="editButtonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="editBeforeInfoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>560</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: blue;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="editCombobox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border-radius: 5px;
	border: 1px solid black;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid blue;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Введите сумму расхода</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="editDateEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
    border-radius: 5px;
	border: 1px solid black;
}</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="beforeLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>До</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>editButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
